src/Classes/Cart.php

```diff
@@ -45,5 +47,39 @@ function checkId($id)
         }
     }
 
+    function emailItems()
+    {
+        $db = Database::getinstance();
```

Comment:
Each of these first three chunks where you're writing queries and reading from the DB could be argued to have their own respective functions to handle it since that way it'd be repeatable and you could use the results elsewhere in the codebase. You don't have to do this, but try to keep it in mind when writing code in terms of flexibility and reusability